admin/server/services.py

```
@@ -582,7 +582,7 @@ def get_config():
                 "config": provider_config,
             }
         except Exception as e:
-            raise AdminException(f"Failed to get sandbox config: {str(e)}")
+            raise AdminException(f"Failed to get sandbox config: {e!s}")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)